util/fibers/detail/scheduler.cc

```diff
@@ -553,11 +555,18 @@ void Scheduler::ScheduleFromRemote(FiberInterface* cntx) {
 void Scheduler::Attach(FiberInterface* cntx) {
   cntx->scheduler_ = this;
 
+  fibers_.push_back(*cntx);
```

Comment:
Please add a test involving  `ProactorBase::Migrate` call which migrates the calling fiber to a differerrent (proactor) thread

Reply:
There's a `ProactorTest.Migrate` test already, do you want to test specifically the combination of migration and the stack traces?